modules/gapi/src/compiler/gcompiler.cpp

```diff
@@ -48,9 +48,17 @@ namespace
 {
     cv::gapi::GKernelPackage getKernelPackage(cv::GCompileArgs &args)
     {
+        auto withAuxKernels = [](const cv::gapi::GKernelPackage& pkg) {
+            cv::gapi::GKernelPackage aux_pkg;
+            for (const auto &b : pkg.backends()) {
+                aux_pkg = combine(aux_pkg, b.priv().auxiliaryKernels());
+            }
+            return combine(pkg, aux_pkg);
```

Comment:
Looks like std::accumulate, you don't even need pass a lambda but can pass `&combine` directly.